### DISCUSSION TITLE
howto control LED and LEDLINK (REFOSS P11)

### DISCUSSION THREAD

**Discussion by dkotte** [arendst/Tasmota#19751] [Q&A] [answered]
Hi all,
I use a refoss p11 plug and wish to control the red-LED.
I am able to use LEDPOWER (and LEDPOWER1) to control the green LED.
I am unable to control the second (RED) LED.
If I change gpio4 from ledlink to LED 2 --- the red LED lights all the time, but I'm unable to control it using "ledpower2 2"
Where is my mistake?

**Comment by barbudor** [comment, marked as answer]
You can't control LedLink. It's a built-in funciton of Tasmota
You can configure the 2nd Led as Relay2 and control it with POWER2 commands

**Reply by sfromis** [reply]
For more information about how you can tweak status LED usage, see https://tasmota.github.io/docs/Lights/#status-leds

Note that many plug-type devices will have a LED which is hardwired to the relay status.

**Reply by dkotte** [reply]
Thanks a lot.
Configuring the LED-GPIOs as RELAY gives full control over the LEDs.
